osu.Game/Screens/Menu/IntroScreen.cs

```diff
@@ -28,11 +28,18 @@ public abstract class IntroScreen : StartupScreen
 
         private Bindable<bool> menuVoice;
 
+        private LeasedBindable<WorkingBeatmap> beatmap;
+
+        public new Bindable<WorkingBeatmap> Beatmap { get => beatmap; }
```

Comment:
i'd rather this didn't exist

Reply:
I don't have a more elegant solution in mind other than shadowing Beatmap bindable as it can't be overriden.